e2e/tests/interchain_accounts/base_test.go

```diff
@@ -81,6 +81,9 @@ func (s *InterchainAccountsTestSuite) TestMsgSendTx_SuccessfulTransfer() {
 		s.StartRelayer(relayer)
 	})
 
+	// Wait for a couple of blocks to ensure channel handshake is complete
+	s.Require().NoError(test.WaitForBlocks(ctx, 5, chainA, chainB), "failed to wait for blocks")
+
```

Comment:
do we need to wait 5 blocks or is this arbitrary?

Reply:
seems like it should be bundled into `s.StartRelayer(relayer)`?

Reply:
yess, that's what I was thinking with https://github.com/cosmos/ibc-go/pull/3408#issuecomment-1547467593. I don't want to merge this pr until I can get these pesky `WaitForBlock` calls probably in `StartRelayer` (carlos mentioned that they might be able to be plugged in `interchaintest` which might be even better).

Reply:
Looked through this yesterday and mostly leaning towards this being an issue with hermes being started with the flag `--full-scan` which is documented as slow (reference https://github.com/informalsystems/hermes/issues/1536). I'm going to change interchain test to have the ability to pass in some startup args so we can control if we want to actually do a full scan. Then, we can test if the `--full-scan` is actually required in our case.

Reply:
Allowing extra flags in interchaintest https://github.com/strangelove-ventures/interchaintest/pull/583

This will allow us to test having hermes start w/o `--full-scan`.

Reply:
Have you tested now if we can remove the waiting if hermes starts without the `--full-scan` flag?

Reply:
I did not want to go through and change the paths again to point to my fork. Locally with one test I checked IIRC it passed when I removed the waiting. The tests in interchaintest also pass w/o me needing to manually pass in `--full-scan` :partying_face: 

I'll be waiting a bit (since I opened the PR end of previous week) to see if the folks at strangelove merge it. If not, I'll just ping them in the channel so we can move forward here too.